zenhappens/src/_includes/_footer.html

```diff
@@ -0,0 +1,25 @@
+    <footer>
+        <div class="footer__container">
+        <div class="footer__logo">
+            <p class="logo--wordmark">Zen Happens</p>
+            <p class="logo--tagline">Massage & Wellness</p>
+            <div class="social__icons">
+                <ul class="footer__social--icons">
+                    <li><a href="https://www.yelp.com/biz/zen-happens-massage-and-wellness-baltimore"><img src="/images/icon-yelp.svg" alt="Yelp icon."></a></li>
+                    <li><a href="https://www.facebook.com/zenhappensincharmcity"><img src="/images/icon-fb.svg" alt="Facebook icon."></a></li>
+                    <li><a href="https://www.instagram.com/zen_happens/"><img class="socialicon__align" src="/images/icon-insta.svg" alt="Instagram icon."></a></li>
+                </ul> 
+            </div>
+        </div>
+        <address class="footer__contact">
+                <p>4337 Harford Rd <br>
+                Baltimore, MD 21214 <br>
+                <a href="tel:+14107460499">410-746-0499</a><br>
+                <a href="mailto:ZenHappens@gmail.com">ZenHappens@gmail.com</a> <br></p>
```

Comment:
You shouldn't need a final line break here. If you're using that for spacing purposes, you should instead use margin or padding.
```suggestion
                <a href="mailto:ZenHappens@gmail.com">ZenHappens@gmail.com</a></p>
```